compiler/tests/testSuite.bal

```diff
@@ -69,22 +108,29 @@ function compileErrorLineNumber(CompileError err) returns int? {
     }
 }
 
-function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo");
+function listSourcesVPO() returns TestSuiteCases|error => listSources("vpo", "./testSuite");
 
-function listSourcesEU() returns TestSuiteCases|error => listSources("eu");
+function listSourcesEU() returns TestSuiteCases|error => listSources("eu", "./testSuite");
 
-function listSources(string initialChars) returns TestSuiteCases|io:Error|file:Error {
+function listSources(string initialChars, string path) returns TestSuiteCases|io:Error|file:Error {
     TestSuiteCases cases = {};
     // JBUG #32615 can't use from-in-from query syntax
-    foreach var dir in check file:readDir("./testSuite") {
+    foreach var dir in check file:readDir(path) {
         if !check file:test(dir.absPath, file:IS_DIR) {
             continue;
         }
         string category = check file:basename(dir.absPath);
         foreach var test in check file:readDir(dir.absPath) {
             string name = check file:basename(test.absPath);
             var [base, ext] = basenameExtension(name);
-            if ext != ".bal" {
+            if check file:test(test.absPath, file:IS_DIR) && ext == ".modules" {
+                TestSuiteCases subCases = check listSources(initialChars, test.absPath);
```

Comment:
Why do you think you need to listSources in *.module?

Reply:
I thought we need to add them to validate whenever we get a test failure it was cause by import not an issue in the imported file itself (because it was tested separately) . Removed